src/sensor.h

```diff
@@ -80,13 +84,10 @@ namespace librealsense
             return {};
         }
 
-        std::shared_ptr<std::map<uint32_t, rs2_format>>& get_fourcc_to_rs2_format_map();
-        std::shared_ptr<std::map<uint32_t, rs2_stream>>& get_fourcc_to_rs2_stream_map();
-
-        rs2_format fourcc_to_rs2_format(uint32_t format) const;
-        rs2_stream fourcc_to_rs2_stream(uint32_t fourcc_format) const;
-
     protected:
+        // Since _profiles is private, we need a way to get the final profiles
```

Comment:
This is `sensor_base`, that is, intended to be inherited from.
Why not make the member protected?

Reply:
The member is pretty advanced (`lazy`) and really should be private, I didn't want to make it accessible.